db/schema.rb

```diff
@@ -429,11 +433,11 @@
 
   create_table "reports", force: true do |t|
     t.integer  "item_id",                    null: false
-    t.string   "item_type",                  null: false
     t.boolean  "reviewed",   default: false
     t.text     "text"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "item_type",                  null: false
     t.integer  "user_id",                    null: false
   end
 
```

Comment:
No actual schema changes in the migration, please don't commit schema.rb changes.